### PR TITLE
first commit of contents

### PR DESCRIPTION
**Pull request by Prit-ranjan**
@adithnaveen

### CODE REVIEW

File: content/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import express from 'express';
+import mongoose from 'mongoose';
+import bodyParser from 'body-parser';
+import routes from './src/routes/content-route';
+
+// variable declaration
+const app = express();
+const PORT = 3000;
```
Comment (adithnaveen) [inline content/index.js]: pls take it from .env

File: content/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+mongoose.Promise = global.Promise;
+mongoose.connect('mongodb://localhost:27017/video-streaming-db', { useNewUrlParser: true });
```
Comment (adithnaveen) [inline content/index.js]: take it from .env

File: content/src/models/content-model.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    title: {
+        type: String,
+        required: "Please enter first name"
+    },
+    
+    hashtag: {
+        type: String,
+        
+    },
+    description: {
```
Comment (adithnaveen) [inline content/src/models/content-model.js]:
```suggestion
    description: String 
```

this is better

File: content/src/models/content-model.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        type: String,
+    },
+    video_link: {
```
Comment (adithnaveen) [inline content/src/models/content-model.js]:
```suggestion
    video_link: String 
```


this is better

File: content/src/models/content-model.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    uploaded_date: {
+        type: Date,
+        default: Date.now
+    },
+    content_type: {
+        type: String,
+        enum: ["sports","comedy","horror","dance"],
+        default: "sports"
+    },
+    language:{
```
Comment (adithnaveen) [inline content/src/models/content-model.js]:
```suggestion
    language:String 
```

this is better  and address other fields too

File: content/src/routes/content-route.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  addContent,
+  getAllContent,
+  getContentByLanguage,
+  getContentById,
+  deleteContentById,
+  home
+} from "../controllers/content-controller";
+
+const routes = (app) => {
+  app.route("/").get(home);
```
Comment (adithnaveen) [inline content/src/routes/content-route.js]: not needed

**Review by adithnaveen — CHANGES_REQUESTED** [review]
pls look at the comments

File: content/index.js (hunk 1 above)
Comment (Prit-ranjan) [reply content/index.js]: done

File: content/index.js (hunk 2 above)
Comment (Prit-ranjan) [reply content/index.js]: done

File: content/src/models/content-model.js (hunk 3 above)
Comment (Prit-ranjan) [reply content/src/models/content-model.js]: done

File: content/src/models/content-model.js (hunk 4 above)
Comment (Prit-ranjan) [reply content/src/models/content-model.js]: done

File: content/src/models/content-model.js (hunk 5 above)
Comment (Prit-ranjan) [reply content/src/models/content-model.js]: done

File: content/src/routes/content-route.js (hunk 6 above)
Comment (Prit-ranjan) [reply content/src/routes/content-route.js]: done

**Review by adithnaveen — APPROVED** [review]
pls merge

**Pull request merged**